toolz/functoolz.py

```diff
@@ -541,3 +541,32 @@ def do(func, x):
     """
     func(x)
     return x
+
+
+@curry
+def flip(func, a, b):
+    """Call the function call with the arguments flipped.
+
+    Parameters
+    ----------
+    f : callable
+        A binary function.
+    a : any
+        The second argument to `f`.
+    b : any
+        The first argument to `f`.
+
+    Returns
```

Comment:
Do you think just leaving the example section and brief description is enough?

I find that I have recently led to numpy-style as a defaul but I agree that it is overkill for simple utils like flip